smoketest/scripts/cli/test_system_option.py

```diff
@@ -67,11 +68,38 @@ def test_performance(self):
         gc_thresh2 = '262000'
         gc_thresh3 = '524000'
 
-        self.cli_set(['system', 'sysctl', 'parameter', 'net.ipv4.neigh.default.gc_thresh1', 'value', gc_thresh1])
-        self.cli_set(['system', 'sysctl', 'parameter', 'net.ipv4.neigh.default.gc_thresh2', 'value', gc_thresh2])
-        self.cli_set(['system', 'sysctl', 'parameter', 'net.ipv4.neigh.default.gc_thresh3', 'value', gc_thresh3])
-
-        self.cli_set(base_path + ['performance', 'throughput'])
+        self.cli_set(
+            [
```

Comment:
Can we please keep this in a single line? Lines of code would explode otherwise. It would be good to consolidate the path using: `base_path = ['system', 'sysctl', 'parameter']` and then simply refer to the variable.

Reply:
done